registration/views.py

```diff
@@ -825,12 +832,19 @@ def form_valid(self, form):
                 form.add_error(field=None, error=error_non_field_names)
             return self.form_invalid(form)
 
-        return super(EmailNextOfKinView, self).form_valid(form)
+        return super(EmailNextOfKinBaseView, self).form_valid(form)
 
     def submit_nok_email(self, data):
         try:
             data_copy = data.copy()
+            data_copy['variant'] = self.variant
             data_copy['from_email'] = self.request.session.get(SESSION_EMAIL, settings.DEFAULT_FROM_EMAIL)
+            body = data_copy['body'] + _(
+                '\n\nYou can register as an organ donor at <a href="https://register.organize.org?reg_source=email-nok'
+                '&variant={variant}">https://register.organize.org</a>.\n').format(**{'variant': self.variant})
+            # Replace new lines with html breaks before submission since user might add new lines
+            body = body.replace('\n', '<br>')
```

Comment:
The reason for replacing the `\n` with `<br>` that at first, we need to display the message in the text area in which the new line is `\n`, but before sending the content to 53 in which we expect to get `html` we need to convert it to `<br>` so it works.
The part above I used `\n` even though its not displayable for user since I want to be consistant when writing new line